packages/malloy-interfaces/thrift/malloy.thrift

```diff
@@ -181,409 +162,342 @@ struct FilterExpressionType {
-  5: required SQLNativeType sql_native_type,
-  6: required DateType date_type,
-  7: required TimestampType timestamp_type,
-  9: required ArrayType array_type,
-  10: required RecordType record_type,
+  1: required StringType string_type;
+  2: required BooleanType boolean_type;
+  3: required NumberType number_type;
+  4: required JSONType json_type;
+  5: required SQLNativeType sql_native_type;
+  6: required DateType date_type;
+  7: required TimestampType timestamp_type;
+  9: required ArrayType array_type;
+  10: required RecordType record_type;
 }
 
 union ParameterType {
-  1: required StringType string_type,
-  2: required BooleanType boolean_type,
-  3: required NumberType number_type,
-  4: required JSONType json_type,
-  5: required SQLNativeType sql_native_type,
-  6: required DateType date_type,
-  7: required TimestampType timestamp_type,
-  9: required ArrayType array_type,
-  10: required RecordType record_type,
-  11: required FilterExpressionType filter_expression_type,
+  1: required StringType string_type;
+  2: required BooleanType boolean_type;
+  3: required NumberType number_type;
+  4: required JSONType json_type;
+  5: required SQLNativeType sql_native_type;
+  6: required DateType date_type;
+  7: required TimestampType timestamp_type;
+  9: required ArrayType array_type;
+  10: required RecordType record_type;
+  11: required FilterExpressionType filter_expression_type;
 }
 
 struct SQLNativeType {
-  1: optional string sql_type,
+  1: optional string sql_type;
 }
 
 enum DateTimeframe {
-  YEAR = 1,
-  QUARTER = 2,
-  MONTH = 3,
-  WEEK = 4,
-  DAY = 5,
+  YEAR = 1;
+  QUARTER = 2;
+  MONTH = 3;
+  WEEK = 4;
+  DAY = 5;
 }
 
 enum TimestampTimeframe {
-  YEAR = 1,
-  QUARTER = 2,
-  MONTH = 3,
-  WEEK = 4,
-  DAY = 5,
-  HOUR = 6,
-  MINUTE = 7,
-  SECOND = 8,
+  YEAR = 1;
+  QUARTER = 2;
+  MONTH = 3;
+  WEEK = 4;
+  DAY = 5;
+  HOUR = 6;
+  MINUTE = 7;
+  SECOND = 8;
 }
 
 struct DateType {
-  2: optional DateTimeframe timeframe,
+  2: optional DateTimeframe timeframe;
 }
 
 struct TimestampType {
-  2: optional TimestampTimeframe timeframe,
+  2: optional TimestampTimeframe timeframe;
 }
 
-/*
-
-Questions:
-- Should `fields`, `sources`, etc. be Map<string, Field> or Field[]
-- Any way to have an "either" type? use unions?
-- How to represent a refinement of a model query
-
-Generate typescript?
-- https://github.com/creditkarma/thrift-typescript
-    - https://www.internalfb.com/diff/D68557062
-*/
-
-union ViewOperation {
-  1: required GroupBy group_by,
-  2: required Aggregate aggregate,
-  3: OrderBy order_by,
-  4: required Limit limit,
-  5: required FilterOperation where,
-  6: required Nest nest,
-  7: required FilterOperation having,
+union ViewStatement {
+  1: required GroupBy group_by;
+  2: required Aggregate aggregate;
+  3: required OrderBy order_by;
+  4: required Limit limit;
+  5: required FilterOperation where;
+  6: required Nest nest;
+  7: required FilterOperation having;
 }
```

Comment:
`DrillOperation` is theoretically going to be added, too. https://github.com/malloydata/malloy/pull/2271

Reply:
We should discuss whether we need a framework for "experimental" features in the "stable" interface?

Reply:
For drill, MToy is happy to have it not be experimental. Discuss later about experimental features.